ibis/backends/tests/test_client.py

```diff
@@ -94,7 +104,18 @@ def test_create_table(backend, con, temp_table, lamduh, sch):
     )
 
     obj = lamduh(df)
-    con.create_table(temp_table, obj, schema=sch)
+    if backend.name() == "flink":
+        con.create_table(
+            temp_table,
+            obj,
+            schema=sch,
+            tbl_properties={
+                "connector": None,
+            },
```

Comment:
same

Reply:
Referring to the [reply above](https://github.com/ibis-project/ibis/pull/6920#discussion_r1372456721).